Source/WebCore/css/CSSPrimitiveValue.cpp

```diff
@@ -945,12 +945,12 @@ double CSSPrimitiveValue::computeNonCalcLengthDouble(const CSSToLengthConversion
         return value * lengthOfViewportPhysicalAxisForLogicalAxis(LogicalBoxAxis::Inline, conversionData.dynamicViewportFactor(), conversionData.style());
 
     case CSSUnitType::CSS_LHS:
-        ASSERT(conversionData.style());
+        ASSERT(conversionData.styleForFontUnits());
         if (conversionData.computingLineHeight() || conversionData.computingFontSize()) {
             // Try to get the parent's computed line-height, or fall back to the initial line-height of this element's font spacing.
-            value *= conversionData.parentStyle() ? conversionData.parentStyle()->computedLineHeight() : conversionData.style()->metricsOfPrimaryFont().lineSpacing();
+            value *= conversionData.parentStyle() ? conversionData.parentStyle()->computedLineHeight() : conversionData.styleForFontUnits()->metricsOfPrimaryFont().lineSpacing();
```

Comment:
Does this actually need to change? We've already checked `conversionData.parentStyle()`, so it seems like the `parentStyle()` inside `styleForRontUnits` would be `nullptr`.  Or maybe this is a different value?